### PR TITLE
Update examples.yml

### PR DESCRIPTION
**Pull request by juntao**
Hide modules path in commands

Signed-off-by: Michael Yuan <michael@michaelyuan.com>

### CODE REVIEW

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -42,14 +42,12 @@ jobs:
     - name: Regular examples
       run: |
         cargo build --target wasm32-wasi --release
-        cd example_js
-        wasmedge --env 'a=1' --env 'b=1' --dir .:. ../target/wasm32-wasi/release/wasmedge_quickjs.wasm hello.js WasmEdge Runtime
+        wasmedge --env 'a=1' --env 'b=1' --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/hello.js WasmEdge Runtime
     
     - name: Image examples
       run: |
         cargo build --target wasm32-wasi --release
```
Comment (L-jasmine) [inline .github/workflows/examples.yml:49]: `cargo build --target wasm32-wasi --release --features=img`

File: .github/workflows/examples.yml [hunk 2]
```diff
@@ -119,8 +116,9 @@ jobs:
         cd example_js/react18_ssr
         npm install
         npm run build
+        cd ../..
         echo '> start wasmedge'
-        nohup wasmedge --dir .:. --dir ./modules:../../modules ../../target/wasm32-wasi/release/wasmedge_quickjs.wasm dist/main.mjs &
+        nohup wasmedge --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/react18_ssr/dist/main.mjs &
```
Comment (L-jasmine) [inline .github/workflows/examples.yml:121]: `wasmedge --dir .:. --dir ./public:./example_js/react18_ssr/public target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/react18_ssr/dist/main.mjs`

this `main.js` will load `public/main.css`

Comment (juntao) [reply .github/workflows/examples.yml:121]: Hmm, do you mean that we should always start the SSR server from its own directory?

Comment (juntao) [reply .github/workflows/examples.yml:121]: Ok. I think I understand it now.

File: .github/workflows/examples.yml (hunk 1 above)
Comment (juntao) [reply .github/workflows/examples.yml]: I added the `img` flag and it now complains that `image.js` cannot find the `bird.png` file. I think the JS should always be able to find files in the same directory as itself. In fact, the ES6 module demo works file -- it was able to find the module JS files in the same directory.

**Pull request merged**